core/relay/src/services/helpers.rs

```diff
@@ -86,18 +86,60 @@ pub fn update_event_subscription_status(
                             request_id: curr_request_id.clone(),
                             message: "No event subscription status set in database".to_string(),
                             event_matcher: fetched_event_sub_state.event_matcher.clone(),
-                            event_publication_spec: fetched_event_sub_state.event_publication_spec.clone(),
+                            event_publication_specs: fetched_event_sub_state.event_publication_specs.clone(),
                         };
                     },
                 }
             } else {
-                target = EventSubscriptionState {
-                    status: event_subscription_state::Status::Error as i32,
-                    request_id: curr_request_id.clone(),
-                    message: message.to_string(),
-                    event_matcher: fetched_event_sub_state.event_matcher.clone(),
-                    event_publication_spec: fetched_event_sub_state.event_publication_spec.clone(),
-                };
+                match message.find("Event subscription already exists with requestId: ") {
```

Comment:
Can we move the string to a separate constants file for easier maintainability?

Reply:
Where should I put this constants file? and should driver also refer to the same constants file to raise the same error message?

Reply:
For now, can we have two separate constants files, one in the relay module and another in the driver module, but both containing the same definitions? We can determine later how to create a common constants file that can be shared across the two modules.

Reply:
okay